mapzen-places-api/src/main/java/com/mapzen/places/api/internal/PeliasFeatureToPlaceConverter.java

```diff
@@ -0,0 +1,60 @@
+package com.mapzen.places.api.internal;
+
+import com.mapzen.pelias.gson.Feature;
+import com.mapzen.places.api.LatLng;
+import com.mapzen.places.api.LatLngBounds;
+import com.mapzen.places.api.Place;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Handles converting Pelias {@link Feature} objects into {@link Place} objects for
+ * {@link PeliasCallbackHandler}.
+ */
+class PeliasFeatureToPlaceConverter {
+
+  private PeliasLayerToPlaceTypeConverter layerConverter;
+
+  /**
+   * Constructor.
+   */
+  PeliasFeatureToPlaceConverter() {
+    layerConverter = new PeliasLayerToPlaceTypeConverter();
+  }
+
+  /**
+   * Construct a {@link Place} object from the given feature.
+   * @param feature
+   * @return
+   */
+  Place getFetchedPlace(Feature feature) {
+    //TODO: fill in missing values
```

Comment:
We need some way to communicate to developers which fields in the `Place` object are currently supported and which are not. Perhaps some combination of the following:

1. Documentation - general documentation in README, /docs folder
2. Javadoc - explicitly call out in Javadoc which methods are not yet supported ex. `Place#getAttributions()`
3. Exception - throw a Runtime exception with message "Sorry, not yet supported" for unsupported fields
4. Return value - have a specific error code returned for fields not yet supported. ex. Strings could be "Not yet supported" and numbers could be `-1`?
5. Comment out unsupported methods - if they don't exist they cannot be called.